rinex/src/epoch/flag.rs

```diff
@@ -10,10 +10,14 @@ pub enum Error {
     UnknownFlag,
 }
 
+#[cfg(feature = "pyo3")]
+use pyo3::prelude::*;
+
 /// `EpochFlag` validates an epoch, 
 /// or describes possible events that occurred
 #[derive(Copy, Clone, Debug)]
 #[derive(PartialEq, Eq, PartialOrd, Ord, Hash)]
+#[cfg_attr(feature = "pyo3", pyclass)]
```

Comment:
@larsnaesbye ,
let's start easy with basic structures that do not have any dependents.

We'll have to solve several issues before we can derive `pyclass`  for `Rinex`.

We can't derive `pyclass`  for `Constellation` because Pyo3 does not allow nested enums like we have for SBAS description for example. Hopefully this is an open topic on their side? i don't know.

Then we need to describe the conversion for "complex" structures. Mainly: all record types and vectorized stuff. Apparently this is doable! but I just don't understand how to do it. An auto derivation method does not seem to exist. I think we need to move on to manual derivation (Trait implementation)

Here [is the conversion guideline](https://pyo3.rs/v0.16.2/conversions/tables.html) we should follow
for Python<->Rust conversion

that would be something like `IntoPyList` for vectors, and  `IntoPyDict` for all record types ?

Reply:
The basics is there, it is now working :) 
An example is attached and will get incremented.
Now we need to tackle the hardest parts 😄

Reply:
> and we probably have to submit a PR to `PyO3` for the nested Enum case

we don't necessarily have to go this far. The immediate problem is the `pyclass` auto derivation does not support nested enums, that means this macro cannot dictate the code in this scenario.
We just have to to the job ourselves.
And it's probably a manual `IntoPyObject` implementation [like these ones](https://github.com/PyO3/pyo3/blob/249c0209fd41336b9e68bae14290e598fcad34ea/src/conversions/chrono.rs)